src/sentry/static/sentry/app/components/fileChange.jsx

```diff
@@ -1,38 +1,31 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import createReactClass from 'create-react-class';
 import Avatar from 'app/components/avatar';
 import IconFileGeneric from 'app/icons/icon-file-generic';
 import Tooltip from 'app/components/tooltip';
-import ApiMixin from 'app/mixins/apiMixin';
 
-const FileChange = createReactClass({
-  displayName: 'FileChange',
-
-  propTypes: {
+class FileChange extends React.PureComponent {
+  static propTypes = {
     filename: PropTypes.string.isRequired,
     authors: PropTypes.array.isRequired,
-  },
-
-  mixins: [ApiMixin],
+  };
 
   getInitialState() {
```

Comment:
Does this need to be in a constructor now?

Reply:
Yes it would. Thanks!